nixos/modules/hardware/wooting.nix

```diff
@@ -2,8 +2,15 @@
 
 with lib;
 {
-  options.hardware.wooting.enable =
-    mkEnableOption (lib.mdDoc "support for Wooting keyboards");
+  options.hardware.wooting.enable = mkOption {
```

Comment:
Please revert this and we also don't want to activate this by default

Reply:
My intention here was to add the following clarification:

> Note that users must be in the "input" group for udev rules to apply.

Is there a better way to do this?

Reply:
Good point about the default. I changed it to disabled.

Reply:
> > Note that users must be in the "input" group for udev rules to apply.

that can be done be overwriting the description

Reply:
Done.